ECHO-COVID19.ipynb

```diff
@@ -538,7 +364,45 @@
    },
    "source": [
     "#### Where are these facilities that exceeded their permits?\n",
-    "Even if, on the whole, there are fewer exceedances, the places that are emitting more pollutants are important to track. Their neighbors are suffering more."
+    "Even if, on the whole, there are fewer exceedances, the places that are emitting more pollutants are important to track.\n",
```

Comment:
> there are fewer exceedances

I'm not sure this is the right conclusion. There are fewer *reported* exceedances, but that's the whole problem the COVID policy is enabling

Reply:
(I see the "if" but it's easy to read as: ok, so they didn't exceed, but..)